third_party/opencl-headers.BUILD

```diff
@@ -0,0 +1,23 @@
+licenses(["notice"]) # KHR licence
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "OpenCL-Headers",
+    hdrs = [
+        "opencl22/CL/cl_d3d10.h",
```

Comment:
are we based of opencl2.2?

Reply:
The ICD loader builds using the 2.2 sources, so we have to use those